jpeg_demo/main/esp32_spi_impl.c

```diff
@@ -104,7 +104,7 @@ void deinit_esp32_spi(){
     assert(ret==ESP_OK);
 }
 
-uint8_t esp32_send_spi(SpiProtocolPacket* spiSendPacket){
+uint8_t esp32_send_spi(char* spiSendPacket){
```

Comment:
That should rather be:
```
uint8_t esp32_send_spi(const char* buffer, size_t size)
```
And 
```
uint8_t esp32_recv_spi(char* buffer, size_t size)
```

Or in SPI case a possibility is also:
```
transfer(const char* send, char* receive, size_t size)
```

Reply:
I just changed to use const char* for now. Switching to a transfer could make sense as well but I don't want to add to much stuff to this one MR.

Reply:
Okay - can you write an issue to `depthai-spi-api` and mark it as TODO/feature/... just so that it doesn't get lost

Reply:
Added issue to depthai-spi-api.